studio/components/layouts/SignInLayout/SignInLayout.tsx

```diff
@@ -77,20 +88,12 @@ const SignInLayout = ({
 
   return (
     <>
-      {IS_PLATFORM && (
-        <Head>
-          <script
-            dangerouslySetInnerHTML={{
-              __html: `window._getReturnToPath = ${getReturnToPath.toString()};if (localStorage.getItem('${STORAGE_KEY}') && !(new URLSearchParams(location.search).has('next'))) {location.replace('${
```

Comment:
This code no longer works because the storage key is set when there's `aal1` level for the user. If the user has MFA (and requires `aal2` to fully function), he'll be ping-ponged between `/dashboard` and `/sign-in`.